certified-connectors/CM Informatik AG/apiDefinition.swagger.json

```diff
@@ -0,0 +1,5246 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CMI",
+    "description": "For low-code based data exchange: Transfer given information from M365 directly into CMI. Transfer your data based on CMI dossier, including tasks and contacts, from M365 to the CMI solution. The CMI Custom Connector enables even more collaboration from both worlds. Their strengths combined into an unbeatable combination. With our basic functionality you can make use of almost every M365 app to create new or update given objects in your CMI solution.",
+    "contact": {
+      "name": "CM Informatik AG",
+      "url": "http://www.cmiag.ch",
+      "email": "support@cmiag.ch"
+    },
+    "version": "v1"
+  },
+  "host": "cmi-p1-appsrv-aadbridge.azurewebsites.net",
+  "basePath": "/api/proxy",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/virtual/httprequest": {
+      "post": {
+        "summary": "Send an HTTP request",
+        "description": "Construct a REST API request to invoke.",
+        "operationId": "HttpRequest",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "header",
+            "name": "X-CMI-TENANT-NAME",
+            "required": true,
+            "type": "string",
+            "x-ms-dynamic-values": {
+              "operationId": "GetCmiTenants",
+              "value-path": "tenantName",
+              "value-title": "tenantName"
+            },
+            "x-ms-summary": "CMI Tenant",
+            "description": "CMI Tenant"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The parameters to this operation passed in the body.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/HttpRequestBodyParameters"
+            },
+            "x-ms-summary": "Parameters"
+          }
+        ],
+        "responses": {
+          "default": {
```

Comment:
@shockstate it is not allowed to have a default response only. please define a proper response. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).

Reply:
Hi @Amjed-Ayoub this is a generic endpoint that just forwards an http request, there is no default response, as that depends in the path and method you chose.
What should we do in this case?
Thanks for your answers

Reply:
@shockstate thank you for the clarification. We might be able to ignore this. I will get back to you on this.